Jenkinsfile-testing

```diff
@@ -0,0 +1,68 @@
+library("tdr-jenkinslib")
+
+def versionTag = "v${env.BUILD_NUMBER}"
+def repo = "tdr-auth-utils"
+
+pipeline {
+  agent {
+    label "master"
+  }
+
+  stages {
+    stage("Run git secrets") {
+      steps {
+        script {
+          tdr.runGitSecrets(repo)
+        }
+      }
+    }
+    stage('Test') {
+        agent {
```

Comment:
Looks like 4 space indentation in this section.

Reply:
Ah good spot, cheers. I've changed it.